2019-11-10 00:59:44,394	INFO	main1	<module>	Line:11	app started
2019-11-10 00:59:44,402	WARNING	_internal	_log	Line:122	 * Debugger is active!
2019-11-10 00:59:44,427	INFO	_internal	_log	Line:122	 * Debugger PIN: 185-856-785
2019-11-10 00:59:44,438	INFO	_internal	_log	Line:122	 * Running on http://192.168.2.210:9100/ (Press CTRL+C to quit)
2019-11-10 00:59:50,217	INFO	_internal	_log	Line:122	192.168.2.210 - - [10/Nov/2019 00:59:50] "GET /ProblemSet/problemanswer/7f3682e0-bf42-4dd7-b03e-4d30c17340be HTTP/1.1" 200 -
2019-11-10 00:59:50,363	INFO	_internal	_log	Line:122	192.168.2.210 - - [10/Nov/2019 00:59:50] "GET /ProblemSet/problemanswer/7f3682e0-bf42-4dd7-b03e-4d30c17340be/undefined HTTP/1.1" 200 -
2019-11-10 01:00:10,488	INFO	_internal	_log	Line:122	192.168.2.210 - - [10/Nov/2019 01:00:10] "GET /ProblemSet/problemanswer/7f3682e0-bf42-4dd7-b03e-4d30c17340be HTTP/1.1" 200 -
2019-11-10 01:00:10,644	INFO	_internal	_log	Line:122	192.168.2.210 - - [10/Nov/2019 01:00:10] "GET /ProblemSet/problemanswer/7f3682e0-bf42-4dd7-b03e-4d30c17340be/undefined HTTP/1.1" 200 -
2019-11-10 01:00:20,076	INFO	_internal	_log	Line:122	 * Detected change in 'C:\\development\\examsystem\\services\\daos\\problemsetDAO.py', reloading
